webapp/channels/src/plugins/registry.ts

```diff
@@ -279,7 +280,7 @@ export default class PluginRegistry {
             pluginId: this.id,
             icon: resolveReactElement(icon),
             action,
-            text,
+            text: text as PluggableText,
```

Comment:
A lot of casts in here, just curious what the types normally should be?

Reply:
I think `PluggableText` is correct here because that seems to be the types used by the code that actually uses this value, but I'm not certain of that because the types expected by the Redux reducers may not match this either. The types passed into these methods aren't really reliable as a source of truth because the plugins that call these APIs have their own definitions of this interface which may also be wrong.

The typing for this part of the code is a bunch of pieces of code that have their own ideas of what's correct passed through a series of pieces of code that accept `any` 😅